docs/DeveloperGuide.adoc

```diff
@@ -789,9 +800,11 @@ These instructions only provide a starting point for testers to work on; testers
 . Updating a transaction's details and reflecting changes in the list.
 
 .. Prerequisites: List all transactions using the `list` command. The first transaction is `1. [-] Laksa ($10.50) [Lunch][Food]`.
-.. Test case: `update i/1 | n/Fried Rice` +
+.. Test case: `update i/1 n/Fried Rice` +
+   Expected: Details of the updated transaction and original value of updated content shown in status message. Corresponding listed transaction reflects changes. Timestamp in the status bar is updated.
```

Comment:
I think we don't display the timestamp in the status bar, perhaps the last sentence could be removed? Same goes for the other points in this section!